src/main/scala/pathy/path.scala

```diff
@@ -178,6 +180,11 @@ object Path {
   def parentDir[B,T,S](path: Path[B,T,S]): Option[Path[B,Dir,S]] =
     peel(path).map(_._1)
 
+  def fileParent[B,S](file: Path[B, File, S]): Path[B, Dir, S] = file match {
```

Comment:
Not in this case as there is a structural difference, a "file" looks like `FileIn(Path[_,Dir,_], FileName)` and the intent of `fileParent` is to return the `Path[_,Dir,_]` bit, which is the parent directory of the file.